grakn/concept/proto/concept_proto_builder.py

```diff
@@ -96,7 +96,7 @@ def value_type(value_type_: ValueType):
 
 
 def iid(iid_: str):
-    return bytes.fromhex(iid_)
+    return bytes.fromhex(iid_[2:])
```

Comment:
Previously, the IID did not include `0x`, which is required for Grakn and Graql to parse it correctly. We added it.

Reply:
@alexjpwalker AFAIR, you're trying to remove '0x' part here? If so, the intention might be clearer if you do:

```suggestion
    return bytes.fromhex(iid_.lstrip('0x'))
```